ChangeLog

```diff
@@ -13,6 +13,10 @@ API Changes
      Therefore, mbedtls_platform_zeroize() is moved to the platform module to
      facilitate testing and maintenance.
 
+Features
+   * Add Key Wrapping algorithm defined by NIST in SP 800-38F. Supports only
+     Key Wrapping (KW) and Key Wrapping with Padding (KWP) modes.
```

Comment:
This needs to mention explicitly that it's only for AES. Also, the second sentence is missing a subject.

> Add support for key wrapping modes based on AES as defined by NIST SP 800-38F algorithms KW and KWP and by RFC 3394.